### PR TITLE
Improve support for older OSes

### PR DESCRIPTION
**Pull request by MortalKastor**
Tentative improvement/fix for compatibility with less than up-to-date OSes

- Use GCC 7 on Linux (as recommended in our own [README](https://github.com/LedgerHQ/lib-ledger-core-node-bindings/blob/master/README.md) :trollface:)
- Use VS C++ 2017 (as recommended by [Microsoft](https://github.com/microsoft/nodejs-guidelines/blob/master/windows-environment.md#environment-setup-and-configuration) 🚎)

### CODE REVIEW

File: .github/workflows/prebuild.yml [hunk 1]
```diff
@@ -21,9 +21,9 @@ jobs:
         run: yarn --frozen-lockfile --ignore-scripts
       - name: fetch libcore
         run: yarn preinstall
-      - name: config yarn
```
Comment (valpinkman) [inline .github/workflows/prebuild.yml]: why remove this ?

Comment (MortalKastor) [reply .github/workflows/prebuild.yml]: `node-gyp` is expecting to have VS 2017 to compile code, but only VS 2019 was available on the `windows-latest` image, so this step was needed to tell it where to find the compiler.

Now that the image has been changed to `windows-2016`, `node-gyp` is able to find VS 2017 on its own, and this step isn't needed anymore.

**Comment by gre** [comment]
sorry i mess up my merge conflict resolution. i put back the initial changes

**Comment by gre** [comment]
@MortalKastor we should get this in, right?

**Comment by MortalKastor** [comment]
> @MortalKastor we should (still) get this in, right?

Yes, we should.

**Pull request merged**